### PR TITLE
Update route53-vpn.yaml

### PR DESCRIPTION
**Pull request by answami-dev**
Updated route53 configuration.

### CODE REVIEW

File: samples/ra-vpn/route53-vpn/route53-vpn.yaml [hunk 1]
```diff
@@ -18,13 +18,13 @@ metadata:
 spec:
   fileName: anyconnect-linux64-4.9.05042-webdeploy-k9.pkg
   s3:
-   bucket: sfcn-s3-bucket
+   bucket: answami-s3
```
Comment (sdbrain) [inline samples/ra-vpn/route53-vpn/route53-vpn.yaml:21]: should we leave this as sfcn-s3-bucket?

File: samples/ra-vpn/route53-vpn/route53-vpn.yaml [hunk 2]
```diff
@@ -34,13 +34,13 @@ metadata:
 spec:
   fileName: anyconnect-win-4.9.05042-webdeploy-k9.pkg
   s3:
-   bucket: sfcn-s3-bucket
+   bucket: answami-s3
```
Comment (sdbrain) [inline samples/ra-vpn/route53-vpn/route53-vpn.yaml:37]: lets have this id agnostic..

**Review by sdbrain — COMMENTED** [review]
Please revert the places where you have answami

**Comment by yosefsaputra** [comment]
Closing this PR since it has been fixed here: https://github.com/CiscoDevNet/sfcn/pull/11

**Pull request closed without merging**